File: ios/HybridMath.swift
class HybridMath : HybridMathSpec {
  var hybridContext = margelo.nitro.HybridContext()
  var memorySize: Int {
    return getSizeOf(self)
  }

  public func add(a: Double, b: Double) throws -> Double {
    return a + b
  }
}

File: android/src/main/java/com/margelo/nitro/math/HybridMath.kt
class HybridMath : HybridMathSpec() {
    override val memorySize: Long
        get() = 0L
  
    override var pi: Double
      get() = Double.PI
  
    override fun add(a: Double, b: Double): Double {
      return a + b
    }
  }